lib/author.rb

```diff
@@ -0,0 +1,35 @@
+require './lib/book'
+
+class Author
+  attr_reader :name, :books
+
+  def initialize(info)
+    @first_name = info[:first_name]
+    @last_name = info[:last_name]
+    @name = "#{info[:first_name]} #{info[:last_name]}"
```

Comment:
Since this is dynamically calculated based off of other instance variables, I'd rather it be a method. The less instance variables, the better. Less to keep track of.